medical_procedure_external/views/medical_procedure_external_request_template.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record
+        model="ir.ui.view"
+        id="medical_procedure_external_request_template_form_view"
+    >
+        <field
+            name="name"
+        >medical.procedure.external.request.template.form (in medical_procedure_external_request)</field>
+        <field name="model">medical.procedure.external.request.template</field>
+        <field name="arch" type="xml">
+            <form>
```

Comment:
You could also add a green ribbon when its state is "final" and a red one when it is cancelled